berkeley-function-call-leaderboard/bfcl/eval_checker/eval_runner.py

```diff
@@ -1,18 +1,136 @@
-from checker import ast_checker, exec_checker, executable_checker_rest
-from custom_exception import BadAPIStatusError
-from eval_runner_helper import *
-from eval_checker_constant import TEST_COLLECTION_MAPPING
-from tqdm import tqdm
 import argparse
-from dotenv import load_dotenv
 
+from bfcl.constant import TEST_COLLECTION_MAPPING, TEST_FILE_MAPPING, VERSION_PREFIX
+from bfcl.eval_checker.ast_eval.ast_checker import ast_checker
+from bfcl.eval_checker.eval_runner_helper import *
+from bfcl.eval_checker.executable_eval.custom_exception import BadAPIStatusError
+from bfcl.eval_checker.executable_eval.executable_checker import (
+    executable_checker_non_rest,
+    executable_checker_rest,
+)
+from bfcl.eval_checker.multi_turn_eval.multi_turn_checker import (
+    multi_turn_checker,
+    multi_turn_irrelevance_checker,
+)
+from bfcl.eval_checker.multi_turn_eval.multi_turn_utils import is_empty_execute_response
+from dotenv import load_dotenv
+from tqdm import tqdm
 
 # NOTE: This file should be run in the `eval_checker` directory
 
 
-def single_executable_file_runner(
-    handler, model_result, prompt, model_name, test_category
+def multi_turn_runner(
```

Comment:
What's a `runner`? Need better naming - but can do this as a follow-up, shouldn't block release  since this might require thorough testing.

Reply:
Will address in future PRs